src/bp/core/botpress.ts

```diff
@@ -236,23 +237,30 @@ export class Botpress {
   }
 
   // @deprecated > 11: bot will always include default pipeline stage
-  private async _ensureBotsDefineStage(bots: Map<string, BotConfig>, stage: sdk.Stage): Promise<Boolean> {
+  private async _ensureBotConfigCorrect(bots: Map<string, BotConfig>, stage: sdk.Stage): Promise<Boolean> {
     let hasChanges = false
     await Promise.mapSeries(bots.values(), async bot => {
+      const updatedConfig: any = {}
+
+      if (!bot.defaultLanguage) {
+        this.logger.warn(`Bot "${bot.id}" doesn't have a default language, which is now required`)
```

Comment:
There is already a big banner on the admin UI, and a red exclamation mark indicating the issue, I tried to keep the warning as small as possible since it's really "last-resort"